docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+  type: NodePort
+  externalTrafficPolicy: Cluster
+  sessionAffinity: ClientIP
+  selector:
+    weblogic.domainUID: DOMAIN_UID
+    weblogic.clusterName: CLUSTER_NAME
+  ports:
+  - name: myclustert3channel
+    nodePort: 30999
+    port: 7999
+    protocol: TCP
+    targetPort: 7999
+```
+
+#### Table of NodePort Attributes
+
+|Attribute|Description|
+|---------|-----------|
+|metadata.name|For this particular use case, the NodePort name can be arbitrary as long as it is DNS compatible. But, as a convention it's recommended to use `DOMAIN_UID-cluster-CLUSTER_NAME-ext`. To ensure the name is DNS compatible, use all lower case and convert any underscores (`_`) to dashes (`-`).|
+|metadata.namespace|Must match the namespace of your WebLogic cluster.|
+|metadata.labels|Optional. It's helpful to set a weblogic.domainUid so that cleanup scripts can locate all Kubernetes resources associated with a particular domain UID.|
+|spec.type|Must be `NodePort`|
+|spec.externalTrafficPolicy|Set to `Cluster` for most use cases. This may lower performance, but ensures that a client that attaches to a Node without any pods that match the `spec.selector` will be rerouted to a node with pods that do match. If set to `Local`, then connections to a particular Node will only route to that Node's pods and will fail if the Node doesn't host any pods with the given `spec.selector`. It's recommended for clients of a `spec.externalTrafficPolicy: Local` NodePort to use a URL that resolves to a list of all nodes such as `t3://mynode1,mynode2:30999` so that a client connect attempt will implicitly try mynode2 if mynode1 fails (alternatively, use a round-robin DNS address in place of `mynode1,mynode2`).|
+|spec.sessionAffinity|Set to `ClientIP` to ensure an HTTP tunneling connection always routes to the same pod, otherwise the connection may hang and fail.|
+|spec.selector|Specify a weblogic.domainUID and weblogic.clusterName to associate the NodePort resource with your cluster's pods. The operator automatically sets these labels on the WebLogic cluster pods that it deploys for you.|
+|spec.ports.name|This name is arbitrary.|
+|spec.ports.nodePort|The external port that clients will use. This must match the external port that's configured on the WebLogic configured channels/network-access-points. By default, Kubernetes requires that this value range from 30000 to 32767.|
+|spec.ports.port and spec.targetPort|These must match the port that's configured on the WebLogic configured channel/network-access-point(s)|
+
+### Security Notes
+
+- With some cloud providers, a load balancer or NodePort may implicitly expose a port to the public Internet. 
+
+- If such a port supports a protocol suitable for WebLogic clients, note that WebLogic allows access to JNDI entries, EJB/RMI applications, and JMS by anonymous users by default.
+
+- You can configure a custom channel with a secure protocol and two-way SSL to help prevent external access by unwanted clients.
+
+
+### References
+
```

Comment:
put a lead in sentence here to explain why you are listing references.. are they just for further reading?